### ISSUE TITLE
OracleRespondTx Vs OracleResponseTx 2

### ISSUE THREAD

**Issue by randomshinichi** [aeternity/aeternity#2799]
## Expected Behavior
An OracleResponseTxJSON should deserialize to an OracleResponseTx.
Or an OracleRespondTxJSON should deserialize to an OracleRespondTx.

Keeping with the 'verb' pattern, I'd suggest OracleRespondTxJSON.
## Actual Behavior
```
"OracleResponseTxJSON": {
      "allOf": [
        {
          "$ref": "#/definitions/GenericTx"
        },
        {
          "$ref": "#/definitions/OracleRespondTx"
        }
      ]
    },
```

## Steps to Reproduce the Problem

  1. download the swagger API file.

**Comment by evdoks** [comment]
@velzevur Can you comment on this one?

**Comment by velzevur** [comment]
Will be fixed in 6.0.0

**Comment by velzevur** [comment]
With the introduction of `oas3`, the `OracleResponseTxJSON` is being removed, hence the discrepancy between the two. The `swagger2` API is being kept for backwards compatibility and the issue will be present there until we depreciate it.
